distribution/src/main/resources/config.yml

```diff
@@ -57,3 +57,14 @@ local-app-file-system:
 mapdb-app-file-system:
     drive-name: Database
     db-file: $HOME/farao.db
+
+rao:
+    default: SearchTreeRao
+
+linear-rao-parameters:
+    max-number-of-iterations: 10
+    skip-linear-rao: false
```

Comment:
The name is quite strange, as you run a linear RAO skipping the linear RAO.
(sorry :s)

Maybe : 
`security-analysis-without-rao` : false
`run-only-security-analysis` : false

Reply:
changed the name to 
security-analysis-without-rao : false